exchanges/stream/websocket_test.go

```diff
@@ -234,6 +344,27 @@ func TestWebsocket(t *testing.T) {
 	if err == nil {
 		t.Fatal("should not be connected to able to shut down")
 	}
+
+	ws.verbose = true
```

Comment:
does this need to remain?

Reply:
Debatable, it adds to the code coverage.

Reply:
It is only covering logging though 😄 but since the logs don't output unless you have -v on, I'll resolve this